docs/guide-hermes-release.md

```diff
@@ -20,14 +32,82 @@ git checkout -b rn/0.76-stable
 git push origin HEAD
 ```
 
-### For stable patch releases
+### Step 2: Cherry-pick
+
+#### For (Legacy) Hermes
+
+> [!Important]
+> If you cutting a release candidate, skip this step
+
+Pick the relevant commits onto that branch. The pick requests should be from `main` and no other branch on Hermes.
+
+Push the picks to the remote branch.
+
+#### For HermesV1
+
+> [!Important]
+> If you cutting a release candidate, skip this step
+
+1. Checkout the `250829098.0.0-stable` branch.
+2. Pick the relevant commits onto that branch. The pick requests should be from `static_h` and no other branch on Hermes.
+3. Push the picks to the remote branch.
+
+> [!Important]
+> If you cherry picked something, you need to bump the patch version of hermes, as shown in [this commit](https://github.com/facebook/hermes/commit/ddd708a85b164d1841c024973d0f6d3fad60a4c2)
+> 1. Open the [`npm/hermes-compiler/package.json`](https://github.com/facebook/hermes/blob/ddd708a85b164d1841c024973d0f6d3fad60a4c2/npm/hermes-compiler/package.json) file
+> 2. Bump the patch number by 1
+> 3. Commit and push.
+
+### Step 3: Publish Tag
+
+This step should be automatic. When a new commit is pushed, we should have a new tag in the Hermes repo.
```

Comment:
This should be automatic, but it's a part of the publish workflow. It doesn't happen on every commit.